select2-parent/select2/src/main/java/org/wicketstuff/select2/Select2MultiChoice.java

```diff
@@ -12,128 +12,118 @@
  */
 package org.wicketstuff.select2;
 
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Collection;
-import java.util.Collections;
-import java.util.List;
-
 import org.apache.wicket.markup.ComponentTag;
 import org.apache.wicket.markup.html.form.FormComponent;
 import org.apache.wicket.model.IModel;
 import org.apache.wicket.util.convert.ConversionException;
+import org.apache.wicket.util.string.AppendingStringBuffer;
 import org.apache.wicket.util.string.StringValue;
 
+import java.util.*;
+import java.util.stream.Stream;
+
 /**
  * Multi-select Select2 component. Should be attached to a {@code <select></select>} element.
  *
- * @param <T>
- *            type of choice object
+ * @param <T> type of choice object
  * @author igor
  */
-public class Select2MultiChoice<T> extends AbstractSelect2Choice<T, Collection<T>>
```

Comment:
Could you please revert white-space changes?
All tabs were replaced with spaces :(

Reply:
Changed it.